opencti-platform/opencti-front/src/utils/Charts.js

```diff
@@ -31,6 +32,7 @@ const toolbarOptions = {
       columnDelimiter: ',',
       headerCategory: 'category',
       headerValue: 'value',
+      // eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
Please do not use those disabling rules. Change the name of the argument instead